src/Serilog.Sinks.Console/Sinks/SystemConsole/Output/TimestampTokenRenderer.cs

```diff
@@ -27,17 +27,22 @@ class TimestampTokenRenderer : OutputTemplateTokenRenderer
     readonly ConsoleTheme _theme;
     readonly PropertyToken _token;
     readonly IFormatProvider? _formatProvider;
+    readonly bool _convertToUtc;
 
-    public TimestampTokenRenderer(ConsoleTheme theme, PropertyToken token, IFormatProvider? formatProvider)
+    public TimestampTokenRenderer(ConsoleTheme theme, PropertyToken token, IFormatProvider? formatProvider, bool convertToUtc)
     {
             _theme = theme;
             _token = token;
             _formatProvider = formatProvider;
-        }
+            _convertToUtc = convertToUtc;
+    }
 
     public override void Render(LogEvent logEvent, TextWriter output)
     {
-            var sv = new DateTimeOffsetValue(logEvent.Timestamp);
+        var timestamp = _convertToUtc
+            ? logEvent.Timestamp.ToUniversalTime()
```

Comment:
Unfortunately `DateTimeOffset` ISO formatting in UTC renders as the unanticipated `T00:00` rather than the more common and compact `Z` notation. You should be able to see this by running the code with `{UtcTimestamp}` and no format string.

`MessageTemplateTextFormatter` gets around this using `.UtcDateTime` and renders the value as a `DateTime` with `DateTimeKind.Utc`:

https://github.com/serilog/serilog/blob/dev/src/Serilog/Formatting/Display/MessageTemplateTextFormatter.cs#L107

Reply:
Thanks for pointing that out, I'll have a look into that and will address it in an update.

Reply:
@nblumhardt, the issue you pointed out is now addressed in the latest update. Rendering `{UtcTimestamp}` is now done as a DateTime.